tests/Microsoft.Identity.Test.Unit/CoreTests/CollectionHelperTests.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.Identity.Test.Unit.CoreTests
     [TestClass]
     public class CollectionHelperTests
     {
-#if NETCOREAPP
+#if NET6_WIN || NET6_0
```

Comment:
Can you replace with `if NET6_OR_HIGHER` ?

Reply:
hmm, this directive doesn't seem to be functional
![image](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/assets/30090357/4645e8d6-eb1d-4953-b1d5-50c71fad5769)

Reply:
It's NET6_0_OR_GREATER